src/Style.hs

```diff
@@ -44,7 +90,7 @@ defaults k = maybe (error "unknown property") id $ M.lookup k vals
                   , (BorderRightWidth, Px 0) -- TODO: replace with medium
                   , (MarginRight, Px 0)
                 ]
-
+-}
```

Comment:
we should probably fix up this function to use the right values (`NumUnit Px 0` instead of `Px 0`)